rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -327,6 +327,16 @@ def test_exec(config, ssh_con):
                                 else:
                                     raise AssertionError("Abort multipart failed")
 
+                        elif config.test_ops.get("upload_type") == "failed_upload":
+                            log.info("Failed object upload")
```

Comment:
Can we update the log.info message with more details on read_only site and upload failure is expected.

Reply:
Done